transmission_interface/src/transmission_parser.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "transmission_interface/transmission_parser.hpp"
+#include "transmission_interface/transmission_info.hpp"
+
+#include <tinyxml2.h>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace
+{
```

Comment:
with the constants being `static`, the unnamed namespace should not be needed.

Reply:
Thanks, I'll remove `static` then. The keyword is ambiguous whether it's regarding visibility and initialization strategies. The anonymous namespace clearly captures that these values shouldn't be visible elsewhere and them being const-everything shoudl be enough to denote initialization.